modules/nodes-nodes-working-master-schedulable.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-nodes-working.adoc
+
+[id="nodes-nodes-working-master-schedulable_{context}"]
+= Configuring master nodes as schedulable
+
+As of {product-title} 4.2, you can configure master nodes to be
+schedulable, meaning that new Pods are allowed for placement on the master
+nodes. By default, master nodes are not schedulable. However, if your cluster
+does not contain any worker nodes, then master nodes are marked schedulable by
+default.
+
+You can allow or disallow master nodes to be schedulable by configuring the
+`mastersSchedulable` field.
```

Comment:
@bergerhoffer @ravisantoshgudimetla Do we need a note here: _If your cluster does not contain any worker nodes, do not configure the masters as unschedulable?_ 
(I'm assuming in such a cluster there are no other nodes than master.)

Reply:
Thanks for the question @mburke5678, I don't know the answer to that. @ravisantoshgudimetla - let me know if something like this is worthwhile to add.

I'm going to go ahead and merge this, but we can always add it in a new PR if needed.